frontend/components/TableContainer/DataTable/_styles.scss

```diff
@@ -320,6 +320,11 @@ $shadow-transition-width: 10px;
           min-width: 100%;
           text-align: left;
         }
+        .w150 {
+          max-width: calc(150px - 48px);
```

Comment:
Do we need a calc here? This is just 102px right? Maybe throw in a comment for why it's 102?

Reply:
I was just matchign 250 above, I can update both of them